Tests/UnitTests/CMakeLists.txt

```
@@ -7,14 +7,16 @@
 add_custom_target(unit_tests)
 
 macro(add_unittest _name _source)
+  set(_options EXCLUDE_FROM_PHONY)
```

```suggestion
  # This option allows to exclude certain tests from the target, i.e. tests that depend on non-Core components
  set(_options EXCLUDE_FROM_PHONY)
```

Maybe a comment on why this option is introduced...